src/main/java/com/springboot/onlinebanking/repository/RoleRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.springboot.onlinebanking.repository;
+
+import com.springboot.onlinebanking.model.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    @Query("SELECT r FROM Role r WHERE r.roleName = :roleName")
+    Optional<Role> getRoleByRoleName(String roleName);
```

Comment:
....getRoleByRoleName( @Param("roleName")String roleName));

I think you have add @Param here.

Reply:
@Sashk52 `@Param` is not required here